understanding/22/focus-not-obscured-minimum.html

```diff
@@ -48,14 +53,100 @@ <h2>Intent of Focus Not Obscured (Minimum)</h2>
+          <!--
+            <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustrations showing an abstract representation of a side panel where the tab ring is restricted to the panel</figcaption>
+          </figure>
+          -->
+         </li>
+          <li><strong>The disclosure expands into an area of the page containing no other content.</strong> Many pages are designed with wide margins, providing significant white space into which new content can be opened. Many chatbots and toasters are designed to 'slide up' into the right unpopulated side of a page. Where authors are careful to ensure content is not obscured at each breakpoint in a responsive design, no obscuring of other operable content need occur.<br/>
+
+           <!-- <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustrations showing an abstract representation of a chat window expanding into empty space on the right side of the screen.</figcaption>
+          </figure>
+         -->
+         </li>
+          <li><strong>When focus leaves the additional content, the additional content is hidden or collapsed.</strong> This is very similar to patterns discussed previously under Non-persistent opened information. A distinguishing factor can be that the user's last point of interaction in the disclosure is preserved (it persists) even though it may be hidden until a user returns. Some trees and left navigation pattterns behave this way. <br/>
+            <!--
+            <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustrations showing a collapsible tree that when expanded shows the prior point of interaction</figcaption>
+          </figure>
+          -->
+         </li>
+       </ul>
+       <p>Where opened content persists and causes existing content to be obscured, it will be at risk of failing this criterion if it prevents users from seeing the item receiving focus. An example of this problem is described and captured in a short video in <a href="https://knowbility.org/blog/2020/accessible-slide-menus#slide-out-navigation-stays-open-while-focus-is-on-content-behind-it">Slide-out navigation stays open while focus is on content behind it</a>.</p>
```

Comment:
i think what this section needs is an example of a persistent disclosure (for instance, a 'teaching tip' or 'info button popup' where the popup content can persist if focus leaves the content (if it can even enter in the first place), but a user can dismiss, or minimize the content by use of the Esc key.

Reply:
Please do raise on Friday, or create a PR, or (if lacking time!) create a quick issue. I'm going to merge to make sure the rest gets in for the transition. Thanks.